project-api/ballerina-projects/src/main/java/io/ballerina/projects/writers/BaloWriter.java

```diff
@@ -29,18 +59,347 @@
  */
 public class BaloWriter {
 
+    private static final String MODULES = "modules";
```

Comment:
We can use `ProjectConstants.MODULES_ROOT` without initializing a new constant

Reply:
Added